__tests__/server/config/env/__snapshots__/runTime.spec.js.snap

```diff
@@ -1,5 +1,11 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`runTime HTTP_ONE_APP_DEV_CDN_PORT normalizes numeric input 1`] = `"env var HTTP_ONE_APP_DEV_CDN_PORT needs to be a valid integer, given \\"0002345a\\""`;
+
+exports[`runTime HTTP_ONE_APP_DEV_PROXY_SERVER_PORT normalizes numeric input: env var HTTP_ONE_APP_DEV_PROXY_SERVER_PORT needs to be a valid integer, given "r00t" 1`] = `"env var HTTP_ONE_APP_DEV_PROXY_SERVER_PORT needs to be a valid integer, given \\"r00t\\""`;
+
+exports[`runTime HTTP_PORT throws if neither of HTTP_PORT or HTTPS_PORT are defined 1`] = `"Either of the \`HTTP_PORT\` or \`HTTPS_PORT\` or \`PORT\` environment variables must be defined"`;
```

Comment:
"Either" means two choices I think, but the message has three
```suggestion
exports[`runTime HTTP_PORT throws if neither of HTTP_PORT or HTTPS_PORT are defined 1`] = `"One of the \`HTTP_PORT\` or \`HTTPS_PORT\` or \`PORT\` environment variables must be defined"`;
```

Reply:
Per webster either can mean 2 or sometimes more than 2.

Reply:
depends on the usage
>Definition of either
>
>adjective
1: being the one and the other of two : EACH
_flowers blooming on either side of the walk_
_plays either instrument well_
2: being the one or the other of two
_take either road_
>
>pronoun
the one or the other
_take either of the two routes_
>
>conjunction
used as a function word before two or more coordinate words, phrases, or clauses joined usually by or to indicate that what immediately follows is the first of two or more alternatives
_can be used either as a guest room or as an office_
>
>adverb
1: LIKEWISE, MOREOVER —used for emphasis after a negative
_not smart or handsome either_
2: for that matter —used for emphasis after an alternative following a question or conditional clause especially where negation is implied
_who answers for the Irish parliament? or army either? — Robert Browning_

https://www.merriam-webster.com/dictionary/either

The conjunction form is the only one that mentions more than 2, and that's after saying 2 which appears to be the more common/preferred usage. I think that's due to [writers ignoring "the rules"](https://www.grammarphobia.com/blog/2017/08/either-neither.html) rather than [common usage](https://ell.stackexchange.com/a/63450):
![ngram usage of "either of the three" and "any of the three" with the former dropping significantly since 1880](https://i.stack.imgur.com/YaBON.png)

I think I read something about formal (2) vs informal/conversational (2 & 2+) rules but can't find that now. IMO it's better for technical messages to adhere to formal rules to avoid ambiguity. 

At the risk of bike-shedding I won't require a change here, but I would ask to strongly consider the point on ambiguity.

Thanks, @anescobar1991 !

Reply:
I was viewing an older diff, this isn't even an issue after the latest changes 🤦‍♂